pkgs/top-level/all-packages.nix

```diff
@@ -5588,6 +5588,10 @@ let
 
   node_webkit = node_webkit_0_9;
 
+  nwjs_0_12 = callPackage ../development/tools/node-webkit/nw12.nix {
```

Comment:
Maybe those package should be on the same file. They are really similar
* node_webkit_0_9
* node_webkit_0_11
* nwjs_0_12